spec/api/v2/auth/auth_spec.rb

```diff
@@ -253,4 +253,50 @@
       end
     end
   end
+
+  describe 'test restrictions' do
```

Comment:
i will say, this should go in separate file.
Auth specs are too huge to read now, and its quite good to split them on files, e.q. file for rbac test, permissions test, auth endpoint itslef, restrictions